packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -3355,6 +3355,24 @@
   "Trigger_Words": "Trigger Words",
   "Triggers": "Triggers",
   "True": "True",
+  "Troubleshoot": "Troubleshoot",
+  "Troubleshoot_Description": "These settings are meant to be enabled only with guidance of the Rocket.Chat Development or Support Teams. Do not touch them if you don't know what your are doing!",
+  "Troubleshoot_Disable_Notifications": "Disable Notifications",
+  "Troubleshoot_Disable_Notifications_Alert": "This setting completly disables the notifications system, sounds, desktop notifications, mobile notifications and emails will stop!",
+  "Troubleshoot_Disable_Presence_Broadcast": "Disable Presence Broadcast",
+  "Troubleshoot_Disable_Presence_Broadcast_Alert": "This setting prevents all instances to send the status changes of the users to their clients keeping all the users with their presence status from the first load!",
+  "Troubleshoot_Disable_Instance_Broadcast": "Disable Instance Broadcast",
+  "Troubleshoot_Disable_Instance_Broadcast_Alert": "This setting prevent the Rocket.Chat instances to send events to the other instances, it may couse syncing problems and misbehavior!",
```

Comment:
```suggestion
  "Troubleshoot_Disable_Instance_Broadcast_Alert": "This setting prevents the Rocket.Chat instances to send events to the other instances, it may cause syncing problems and misbehavior!",
```